app.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "auspice.us",
+  "scripts": {},
+  "env": {
+    "HOST": {
+      "required": true
```

```suggestion
    "HOST": {
      "description": "For Auspice server; see https://docs.nextstrain.org/projects/auspice/en/stable/server/index.html#deploying-via-heroku",
      "value": "0.0.0.0"
```

This is currently configured twice: once for review apps and separately for the production app. Since this value isn't secretive, it can be set here to have one central source of truth and also a doc link to go with it.

---

+1